diff --git a/jam2/JAM.cxx b/jam2/JAM.cxx
index 1b2abdd..af48d54 100644
--- a/jam2/JAM.cxx
+++ b/jam2/JAM.cxx
@@ -54,9 +54,9 @@ JAM::JAM(string xmldir, bool printBanner)
 
   double versionNumberXML = pythia->parm("Pythia:versionNumber");
   cout << "# xmlPath= "<< pythia->word("xmlPath") << " Pythia8 version "<< versionNumberXML << std::endl;
-  if(versionNumberXML != 8.315) {
+  if(versionNumberXML < 8.312) {
     cout << "wrong version of Pythia8 is used " << versionNumberXML
-      << " current version of JAM uses pythia8 version 8.315"<< std::endl;
+      << " current version of JAM uses pythia8 version >= 8.312"<< std::endl;
     exit(1);
   }
 
